test/e2e/kubectl.go

```diff
@@ -583,7 +583,6 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			_, err := framework.RunHostCmd(ns, simplePodName, "/kubectl get pods --token=invalid --v=7 2>&1")
 			framework.Logf("got err %v", err)
 			Expect(err).To(HaveOccurred())
-			Expect(err).To(ContainSubstring("User \"system:anonymous\" cannot list pods in the namespace"))
```

Comment:
Upon testing, I need to figure out something to do with that line as well (initial was visual inspection, and I didn't catch the 403/401). This test appears to be passing against GCE, which means the error returned is brittle based on auth plugin.